monitoring/mongodb/dashboard.py

```diff
@@ -277,31 +551,57 @@ def mongodb_state_timeline(title, expr, description="", mappings=None, **kwargs)
     unit=UNITS.BYTES,
     targets=[
         Target(
-            expr=f'sum(mongodb_dbstats_indexSize{{{DATABASE_FILTER_USER}, job=~"$job", rs_state="{RS_STATE_PRIMARY}", namespace="{NAMESPACE}"}})',
+            expr="sum("
+            + Metrics.DBSTATS_INDEX_SIZE(
+                DATABASE_FILTER_USER, f'rs_state="{RS_STATE_PRIMARY}"'
```

Comment:
parameter should be passed as `kwargs` when possible (may be others like this):
```suggestion
                DATABASE_FILTER_USER, rs_state=RS_STATE_PRIMARY
```